chore(FullPageError): add docs and fix example (#5559)
carbon-design-system/ibm-products: docs/guides/v2.md

| EmptyState                | View changes [here](#emptystate)       |
| ExportModal               | No API changes.                        |
| ExpressiveCard            | View changes [here](#expressivecard)   |

> | HTTPError403              | View changes [here](#fullpageerror)    |
> | HTTPError404              | View changes [here](#fullpageerror)    |
> | HTTPErrorOther            | View changes [here](#fullpageerror)    |

Reviewer: Since HTTPError actually still exist also will be deprecated in the next version, maybe something more like ...

```suggestion
| HTTPError403              | No API changes. New version [here](#fullpageerror) |
| HTTPError404              | No API changes. New version [here](#fullpageerror) |
| HTTPErrorOther            | No API changes. New version [here](#fullpageerror) |
```

- `pictogram` can now be either a function or object.

### FullPageError

> - `HTTPError403`,`HTTPError404`, `HTTPErrorOther` have been deprecated, and is
>   been replaced by `FullPageError`

Reviewer: ```suggestion
- `HTTPError403`,`HTTPError404`, `HTTPErrorOther` are deprecated and will
  be removed in the next major version. Please migrate to `FullPageError`.
```

- `errorCodeLabel` --> `label`

> - `links` prop is removed and they need to be passed as children instead of a
>   prop

Reviewer: ```suggestion
- `links` prop is removed; links can be passed as children instead
```
